noxfile.py

```diff
@@ -16,6 +16,7 @@
 PYBAMM_ENV = {
     "SUNDIALS_INST": f"{homedir}/.local",
     "LD_LIBRARY_PATH": f"{homedir}/.local/lib",
+    "PIP_NO_BINARY": "scikits.odes",
```

Comment:
This ensures that `scikits.odes` is always installed from a source distribution. Though we can remove this if needed since no wheels for it exist on PyPI.